src/core/Model/StackTraceInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scheb\Tombstone\Core\Model;
+
+/**
+ * @extends \IteratorAggregate<array-key, StackTraceFrame>
+ * @extends \ArrayAccess<array-key, StackTraceFrame>
+ */
+interface StackTraceInterface extends \IteratorAggregate, \ArrayAccess
```

Comment:
there is an odd with @impliments annotation I've had to extend the interface to satisfy error 

https://github.com/phpstan/phpstan/discussions/5484